fastlane/lib/fastlane/actions/create_itunes_connect_app.rb

```diff
@@ -4,7 +4,7 @@ module SharedValues
       PRODUCE_APPLE_ID = :PRODUCE_APPLE_ID
     end
 
-    class CreateItunesConnectAppAction < Action
+    class CreateAppOnlineAction < Action
```

Comment:
`Online` here means iTunesConnect right? Is having this be `CreateAppOnITunesConnect` too verbose?

Reply:
Check out @milch 's comments from about a week ago - his point was that `produce` creates both iTC and DeveloperPortal apps. I don't love this "online" naming style, but it is a sticky problem.

Reply:
Yeah it's not a good name, I'd prefer something more verbose if it's more expressive. `create_app_on_itunes_connect_or_apple_developer_portal` might be a bit *too* verbose, though 😄

Reply:
Fortunately, creating these aliases (and keeping old ones around) is pretty easy & cheap, so we can always create a new alias later (once we have become smarter and can think of a better name 🤣 ).

Reply:
got it. this makes sense, sorry for missing that before!